oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/VersionGCRecommendations.java

```diff
@@ -428,6 +428,18 @@ public GCResult(boolean ignoreGC, TimeInterval gcScope) {
     }
 
     enum GcType {
-        RGC, DGC
+        RGC("Revision GC"), // revision GC
+        FGC("Full GC"); // full GC
+
+        private final String name;
+
+        GcType(String name) {
+            this.name = name;
```

Comment:
noticed that this changes a [log.warn|https://github.com/apache/jackrabbit-oak/blob/19ee59c6f484c272ffbc04e58c8ba254d7c6da9e/oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/VersionGCRecommendations.java#L408] so could influence down-stream automation that greps for the existing string (RGC vs "Revision GC")

Reply:
if we want this, perhaps there is a way to keep that log.warn intact, eg append this new `name` at the end and keep the original one at `RGC` .. not sure ...

Reply:
but maybe it's fine :)